README.md

```diff
@@ -109,6 +109,10 @@ Start a cluster by running:
 
 `minikube start`
 
+Access Minikube Dashboard UI:
```

Comment:
technically, this is a Kubernetes dashboard. How about:

"Access the Kubernetes dashboard UI"?

Reply:
For accessing Kubernetes Web UI (Dashboard), there is another process, in which we first install Kubernetes UI and then access it. While Minikube dashboard is easy to start. Just:
"minikube dashboard"

Reply:
This command is just a short-cut to start the Kubernetes dashboard UI inside of minikube.

Do you mind changing the text to say "Access the Kubernetes dashboard within minikube"?

Thanks!

Reply:
Yes. You are right. It is basically a Kubernetes Dashboard Addon.

Reply:
@AbdullahBM please address the changes @tstromberg  suggested rest lgtm.

Reply:
Done. Please check it @tstromberg.